grpc/test_support/src/internal/mod.rs

```diff
@@ -36,6 +37,14 @@ pub const AUCTION_INSTALL_CONTRACT: &str = "auction_install.wasm";
 pub const DEFAULT_VALIDATOR_SLOTS: u32 = 5;
 pub const DEFAULT_AUCTION_DELAY: u64 = 3;
 pub const DEFAULT_LOCKED_FUNDS_PERIOD: EraId = 15;
+/// Default round seigniorage rate represented as a fractional number.
+///
+/// Annual issuance: 2%
+/// Minimum round exponent: 14
+/// Ticks per year: 31536000000
```

Comment:
Is this calculated for the minimum round exponent? B/c number of ticks will keep changing.

Reply:
It's the number of _milliseconds_ per year.
If the round length is 2<sup>14</sup> milliseconds, there are 31536000000 / 2<sup>14</sup> _rounds_ per year.